### ISSUE TITLE
Move 'Search Database' option to top of list

### ISSUE THREAD

**Issue by vanaukenk** [geneontology/simple-annoton-editor#84]
For the options on the right-hand side of each entry box in the form, it would be good to move the 'Search Database' option to the top of the list, as it's one of the most frequently used operations.

Thx!

**Comment by tmushayahama** [comment]
@vanaukenk yes, currently it is here on the Form 2.0 .What category should I give to the dangling operations on top ("clear values", "root term") so that it flows smoothly. If search is on top then clear values and root term will seem like they are in search category

![image](https://user-images.githubusercontent.com/2273929/62167973-43674700-b2d9-11e9-8726-86d576146cd1.png)

**Comment by pgaudet** [comment]
Out of date.
